src/modules/UploadModule.ts

```diff
@@ -1,16 +1,19 @@
 import {
   Module,
   Mutation,
-  VuexModule} from 'vuex-module-decorators'
+  VuexModule,
+  getModule } from 'vuex-module-decorators'
 import FfaProcessModule from '../interfaces/vuex/FfaProcessModule'
 import { ProcessStatus } from '../models/ProcessStatus'
-import FileTypeHelper from '../util/FileHelper'
 import FileHelper from '../util/FileHelper'
+import MetaMaskModule from './MetaMaskModule'
+import { Errors } from '../util/Constants'
+import { keccak256 } from 'js-sha3'
```

Comment:
no, you shouldn't need another lib. Web3 has keccak

Reply:
ugh.  it's such a pain in the ass to get to their keccak.